lib/board.rb

```diff
@@ -0,0 +1,43 @@
+# This class is responsible to keep the game board updated
+class Board
+  attr_accessor :board
+  def initialize(player1, player2)
+    @board = { '1' => ' ', '2' => ' ', '3' => ' ',
+               '4' => ' ', '5' => ' ', '6' => ' ',
+               '7' => ' ', '8' => ' ', '9' => ' ' }
```

Comment:
- [ ] **OPTIONAL** As a recommendation, mark the options with a number so the user knows to what space they are referring to. It's for clarity

Reply:
We prefer not to do that.